lib/watir/rspec/version.rb

```diff
@@ -1,5 +1,5 @@
 module Watir
   class RSpec
-    VERSION = "1.1.2"
+    VERSION = "1.1.4"
```

Comment:
Please do not change version in pull requests - i'll do it myself when it's time to release. I'm also trying to follow [Semantic Versioning](http://semver.org) and these changes include backward incompatible changes which mean that the major version should probably be increased.